maths/armstrong_numbers.py

```diff
@@ -0,0 +1,48 @@
+""" An Armstrong number is a number that is equal to the sum of the cubes of its digits.
+For example, 370 is an Armstrong number because 3*3*3 + 7*7*7 + 0*0*0 = 370.
+An Armstrong number is often called Narcissistic number.
+"""
+
+
+def armstrong_number(n):

```

Comment:
Please remove the print() statements from the algorithm itself and just have the function return a bool.  True/False

Put your print() statements in the code that calls the function.

Reply:
Done! :)